client/src/redux/feats/globalSlice/globalSlice.ts

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
```

Comment:
**Question** ❓  - Why was the `timeSlotSlice.ts` replaced with the `globalSlice.ts`?

Reply:
I added global toast into the store. Since timeslots is simply used for read only, i figured we can recycle that file into a general slice

Reply:
> I added global toast into the store. Since timeslots is simply used for read only, i figured we can recycle that file into a general slice

I see. Good idea! 👍